src/components/Landing/index.tsx

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable jsx-a11y/img-redundant-alt */
+import React from "react"
+import PhotoAnalysis from "../PhotoAnalysis"
+
+interface Props {
+  title: string
+  subtitle?: string
+}
+
```

Comment:
In this file, I just pulled in the Example component and then stopped using the Example component in the main index page